staff/pink/ConnectTaskApi/TaskApp-pug/src/api/index.js

```diff
@@ -0,0 +1,68 @@
+const nodeFetch = require('node-fetch')
+
+/**
+ * Task App API client.
+ *
+ * @version 1.0.0
+ */
+var TaskApi;
+(function () {
+    "use strict";
+
+    function call(url) {  
+        
+       return nodeFetch(url).then(res => {
+          
+           return res.json()} )
+    }
+
+    function callWithMethodJson(url, methodProvided, bodyProvided) {
```

Comment:
Considerar en próximos proyectos poder utilizar Axios os puede simplificar el código.

    Make XMLHttpRequests from the browser
    Make http requests from node.js
    Supports the Promise API
    Intercept request and response
    Transform request and response data
    Cancel requests
    Automatic transforms for JSON data
    Client side support for protecting against XSRF


+info: https://medium.com/@shahata/why-i-wont-be-using-fetch-api-in-my-apps-6900e6c6fe78